molgenis-compute-ui/src/main/java/org/molgenis/compute/ui/model/JobStatus.java

```diff
@@ -41,9 +42,10 @@ public boolean isFinalState()
 		}
 
 		@Override
+		//FIXME temp
 		public EnumSet<JobStatus> getStateTransitions()
 		{
-			return EnumSet.of(RUNNING);
+			return EnumSet.of(RUNNING, COMPLETED, FAILED, CANCELLED);
```

Comment:
status transition from RUNNING to CREATED is not allowed. Allowed transitions [COMPLETED, FAILED, CANCELLED]